tmux-cc/src/lib.rs

```diff
@@ -264,8 +277,17 @@ fn parse_line(line: &str) -> anyhow::Result<Event> {
         Rule::layout_change => {
             let mut pairs = pair.into_inner();
             let window = parse_window_id(pairs.next().unwrap())?;
-            let (layout_id, width, height) = parse_window_layout(pairs.next().unwrap())?;
-            Ok(Event::LayoutChange { window, layout_id, width, height })
+            let layout = pairs.next().and_then(parse_window_layout).unwrap();
+            let visible_layout = pairs.next().and_then(parse_window_layout);
+            let raw_flags = pairs
+                .next()
+                .and_then(|r| Option::Some(r.as_str().to_owned()));
```

Comment:
I think you can simplify this:

```suggestion
                .map(|r| r.as_str().to_owned());
```